### ISSUE TITLE
[imgflip.com] need example with use of boxes

### ISSUE THREAD

**Issue by Tokol** [ffflorian/api-clients#573]
tried to used boxex like this:
imgflip.api
            .captionImage({
              username: 'test',
              password: 'test',
              template_id: 289894455,
             boxes:[
                {
                    "text": "One does not simply",
                    "x": 10,
                    "y": 10,
                    "width": 548,
                    "height": 100,
                    "color": "#ffffff",
                    "outline_color": "#000000"
                }
             ]
              // ...
            })
            .then(response => {
              //
              console.log(response);

            });

getting error  response :
{
  success: false,
  error_message: 'Invalid box format. Each box in the boxes array must be an object containing the attributes text, x, y, etc.'
}

**Comment by ffflorian** [comment]
Hi @Tokol, here is the documentation for `Box`: https://ffflorian.github.io/api-clients/packages/imgflip.com/interfaces/_interfaces_.box.html

**Comment by Tokol** [comment]
can you elaborate please! in my code how to implement it

**Comment by ffflorian** [comment]
Your code is already correct - there was a bug in the API client. Please try again with the latest version of my API client. Thanks!

**Comment by Tokol** [comment]
only text is working, x, y and color properties not working

**Comment by ffflorian** [comment]
Thanks for testing @Tokol! I fixed another bug and with [version 3.0.4](https://github.com/ffflorian/api-clients/releases/tag/imgflip.com@3.0.4) and this code:

```ts
import {Imgflip} from 'imgflip.com';

new Imgflip().api
  .captionImage({
    username: 'my-username',
    password: 'my-password',
    template_id: 61579,
    boxes: [
      {
        text: 'One does not simply',
        x: 10,
        y: 10,
        width: 548,
        height: 100,
        color: '#ffffff',
        outline_color: '#000000',
      },
      {
        text: 'Make custom memes on the web via imgflip API',
        x: 10,
        y: 225,
        width: 548,
        height: 100,
        color: '#ff0000',
        outline_color: '#000000',
      },
    ],
  })
  .then(response => console.log(response));
```

I get the following response:
```
{
  success: true,
  data: {
    url: 'https://i.imgflip.com/4t11y0.jpg',
    page_url: 'https://imgflip.com/i/4t11y0'
  }
}
```

![image](https://user-images.githubusercontent.com/5497598/103998712-8864fd80-519c-11eb-8e9c-538a226f2f97.png)

Does it work for you, too?

**Comment by Tokol** [comment]
Yes, it works!
